### PR TITLE
adding postgrest to ubuntu docs

### PR DESCRIPTION
**Pull request by adishM98**
adding postgrest installation guide to ubuntu docs

### CODE REVIEW

File: docs/docs/contributing-guide/setup/ubuntu.md [hunk 1]
```diff
@@ -24,6 +24,11 @@ Follow these steps to setup and run ToolJet on Ubuntu. Open terminal and run the
     sudo apt install postgresql postgresql-contrib
     sudo apt-get install libpq-dev
     ```
+    
+    1.3 Install PostgREST
```
Comment (akshaysasidrn) [inline docs/docs/contributing-guide/setup/ubuntu.md:28]: Can put as (optional)

And below we can say - Required only if TooljetDb is used.

Comment (adishM98) [reply docs/docs/contributing-guide/setup/ubuntu.md]: I have added the changes.

File: docs/docs/contributing-guide/setup/ubuntu.md [hunk 2]
```diff
@@ -24,6 +24,15 @@ Follow these steps to setup and run ToolJet on Ubuntu. Open terminal and run the
     sudo apt install postgresql postgresql-contrib
     sudo apt-get install libpq-dev
     ```
+    
+    1.3 Install PostgREST (optional)
+    
+    :::info 
+    Required only if TooljetDb is being used.
```
Comment (akshaysasidrn) [inline docs/docs/contributing-guide/setup/ubuntu.md:31]: specify version please

Comment (adishM98) [reply docs/docs/contributing-guide/setup/ubuntu.md]: added the specify version under info.

**Review by akshaysasidrn — APPROVED** [review]

**Review by akshaysasidrn — APPROVED** [review]

**Pull request merged**